example/src/main/java/org/apache/rocketmq/example/quickstart/Producer.java

```diff
@@ -94,7 +94,8 @@ public static void main(String[] args) throws MQClientException, InterruptedExce
                 /*
                  * if you want to get the send result in a asynchronize way, you can use this send method
                  * {@code
-                 *
```

Comment:
It is better to add the suggestions in AsyncProduer

Reply:
thx, I'will modify it.